### PR TITLE
refactor: handle key down method

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: src/table/utils/__tests__/handle-key-press.spec.js [hunk 1]
```diff
@@ -142,6 +263,150 @@ describe('handle-key-press', () => {
     });
   });
 
+  describe('headHandleKeyPress', () => {
```
Comment (haxxmaxx) [inline src/table/utils/__tests__/handle-key-press.spec.js:266]: Are these lines only moved, no changes?

File: src/table/utils/handle-key-press.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ export const handleTableWrapperKeyDown = ({
 }) => {
   if (isCtrlShift(evt)) {
     preventDefaultBehavior(evt);
+    // ctrl + shift + left/right arrow keys are used to navigate the table pages
```
Comment (haxxmaxx) [inline src/table/utils/handle-key-press.js:22]:
```suggestion
    // ctrl + shift + left/right arrow keys are used to go to previous/next page
```

File: src/table/utils/__tests__/handle-key-press.spec.js (hunk 1 above)
Comment (LiKang6688) [reply src/table/utils/__tests__/handle-key-press.spec.js:266]: some changes, as headHandleKeyPress = (
  evt,
  rootElement,
  cellCoord,
  column,
  changeSortOrder,
  layout,
  isAnalysisMode,
  setFocusedCellCoord
) is updated to be headHandleKeyPress = ({
  evt,
  rootElement,
  cellCoord,
  column,
  changeSortOrder,
  layout,
  isAnalysisMode,
  setFocusedCellCoord
})

Comment (haxxmaxx) [reply src/table/utils/__tests__/handle-key-press.spec.js:266]: ah ok great

**Review by haxxmaxx — APPROVED** [review]
please merge this right away so I can take it into account into my PRs

**Comment by LiKang6688** [comment]
> please merge this right away so I can take it into account into my PRs

Yes, done

**Pull request merged**